DependencyInjection/Configuration.php

```diff
@@ -72,6 +72,8 @@ public function getConfigTreeBuilder()
                 ->scalarNode('autodetect_theme')->defaultFalse()->end()
                 ->booleanNode('cache_warming')->defaultTrue()->end()
                 ->booleanNode('load_controllers')->defaultTrue()->end()
+                ->booleanNode('assetic_integration')->defaultFalse()->end()
```

Comment:
i think its good to keep it off. did you update the known issues section in the README?